<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitlePage</class>
 <widget class="QWidget" name="TitlePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="journalEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Journal</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="titleEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Conference</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KLineEdit" name="conferenceEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="yearEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="titleButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="yearButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="journalButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="conferenceButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>yearEdit</tabstop>
  <tabstop>journalEdit</tabstop>
  <tabstop>conferenceEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
